server/bingo/models.py

```diff
@@ -1,3 +1,77 @@
-from django.db import models # noqa
+from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
 
-# Create your models here.
+
+class UserManager(BaseUserManager):
+
+    def create_user(self, username, email, password=None):
+
+        user = self.model(
+            username=username,
+            email=self.normalize_email(email)
+        )
+
+        user.set_password(password)
+        user.save(using=self.db)
+        return user
+
+    def create_superuser(self, username, email, password=None):
+        user = self.create_user(
+            username=username,
+            email=email,
+            password=password
+        )
+        user.is_admin = True
+        user.save(using=self.db)
+        return user
+
+
+class User(AbstractBaseUser):
```

Comment:
Oops, I forgot to save this in my above review, but I had also asked you to inherit from ``PermissionsMixin`` here: https://docs.djangoproject.com/en/5.1/topics/auth/customizing/#custom-users-and-permissions